doc/source/contributing_docs.rst

```diff
@@ -169,52 +273,65 @@ environmental variable like this::
+If you have issues building your documentation locally, opening a pull request
+creates preview documentation on ReadTheDocs, which you can use to check renders.
+We believe it is best to open PRs early and often, so that we can catch issues early!
 
-.. code-block:: bash
 
-    # the first time
-    git checkout -b gh-pages
-    git merge origin/my-new-branch
+Optional steps and tips
+=======================
 
-Fix any merge conflicts that arise. Then edit ``UserGuide/doc/source/conf.py`` and change the URL of the site, which is set to ``site_url = "https://www.mdanalysis.org/UserGuide"``. Change it to your personal site, e.g. ::
+The below sections are optional, but may be helpful for more advanced users.
 
-    site_url = "https://www.my_user_name.github.io/UserGuide"
+Using pre-commit hooks
+""""""""""""""""""""""
 
+Manually editing files can often lead to small inconsistencies: a whitespace here, a missing carriage return there.
+A tool called `pre-commit <https://pre-commit.com/>`_ can be used to automatically fix these problems, before a git commit is made.
+To enable the pre-commit hooks, run the following:
 
-Now you can build your pages with the ``make github`` macro in the ``UserGuide/doc/`` directory, which builds the files and copies them to the top level of your directory.
+    .. code-block:: bash
 
-.. code-block:: bash
+        pre-commit install
 
-    make github
+To perform the pre-commit checks on all the files, run the following:
 
-You should be able to open one of these new HTML files (e.g. ``UserGuide/index.html``) in a browser and navigate your new documentation. Check that your changes look right. If they are, push to your `gh-pages` branch from the ``UserGuide/`` directory.
+    .. code-block:: bash
 
-.. code-block:: bash
+        pre-commit run --all-files
 
-    git add .
-    git commit -m 'built my-new-branch'
-    git push -f origin gh-pages
+To remove the pre-commit hooks from your .git directory, run the following:
 
-On GitHub, navigate to your fork of the repository and go to **Settings**. In the **GitHub Pages** section, select the "gh-pages branch" from the **Source** dropdown. Check that your website is published at the given URL.
+    .. code-block:: bash
 
-.. image:: images/gh-pages-settings.png
+        pre-commit uninstall
 
-For each time you add changes to another branch later, just merge into gh-pages and rebuild.
 
-.. code-block:: bash
 
-    git checkout gh-pages
-    git merge origin/my_branch
-    cd doc/
-    make github
+Then :ref:`create a pull request <create-a-pull-request>`.
+The only test run for the User Guide is: that your file compile into HTML documents without errors.
+As usual, a developer will review your PR and merge the code into the User Guide when it looks good.
```

Comment:
Repetition of the section above. Is this intentional?

Reply:
No, oops -- removed as an accidental copy-paste. Thanks for the catch!